src/day4.scala

```diff
@@ -0,0 +1,55 @@
+// using scala 3.0.2
+
+package day4
+
+import scala.io.Source
+
+@main def run(): Unit =
```

Comment:
You'll have to follow the format of `part1` and `part2` with 2 overloads each that the other solutions use. This is necessary to integrate in the website afterward.

Reply:
I discussed it with @sporarum and we concluded that it was easier to solve both parts in a single function. 

For the website part we will have a single box and button that will print the solution of both parts.